### ISSUE TITLE
Dragons wont spawn in specified dimensions/biomes

### ISSUE THREAD

**Issue by CreatureHorns** [Alex-the-666/Ice_and_Fire#4898]
None of the dragons, roosts or dens are generating in the dimensions/biomes specified

This is the biome config file i have for fire dragons
![image](https://github.com/AlexModGuy/Ice_and_Fire/assets/103784957/988f3bff-0b26-46d5-af70-c479b15c36f7)
![image](https://github.com/AlexModGuy/Ice_and_Fire/assets/103784957/8b3c2e8d-bee5-44ac-8e2b-2c7c773cb42f)

The infinite abyss first and second layer dimension is whitelisted, but they still will not spawn at all I've tried following the region wiki page but I'm not able to figure out if I'm doing anything wrong. Same issue with thunder and ice dragon spawning aswell

**Comment by TheBv** [comment]
Please take a look at this : https://github.com/AlexModGuy/Ice_and_Fire/wiki/How-to-specify-the-regions-structures-and-mobs-can-be-spawned
TLDR: Split the infinite abyss biome values apart into their own `[ ]` brackets
